### PR TITLE
Fix #841: Easy Setup and License sections swapped

### LINKED ISSUE #841: [website] [SEO]: switch "easy setup" and "license" in "Features" table
image

While many people care a lot about which license their linters use (like this linter's developer), when it comes to people just googling for a good tool, I think how much work it takes to get up and running is a faster selling point

### PR DESCRIPTION
**Pull request by redconOne**
This fixes issue [#841](https://github.com/quick-lint/quick-lint-js/issues/841)

"easy setup" and "license" under the "Features" section were swapped

### CODE REVIEW

**Comment by redconOne** [comment]
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).

**Comment by redconOne** [comment]
recheck

File: website/public/index.ejs.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
         border-collapse: collapse;
         width: 100%;
       }
+
```
Comment (vegerot) [inline website/public/index.ejs.html:16]: nit: all these formatting changes make this diff difficult to read.  Please revert formatting changes or put them in a separate commit

Comment (strager) [reply website/public/index.ejs.html:16]: Must fix: I agree. Why are there so many formatting changes? Undo them.

Comment (redconOne) [reply website/public/index.ejs.html:16]: Sorry, I initially didn't make those changes. After I finished the working on the issue I followed the [contributing guidelines](https://github.com/quick-lint/quick-lint-js/blob/1d71c38fb3ac3483ef97c65242466d9250763fd4/docs/CONTRIBUTING.md) and ran 
'yarn install'
'yarn fmt'
and it adjusted the file from there. 

So to be clear, I should undo the changes from the 'yarn fmt' command and resubmit?

Comment (strager) [reply website/public/index.ejs.html:16]: That's odd. When I run `yarn fmt` on master, I get no changes at all. Are you sure something else (like your editor) didn't cause those changes?

Comment (redconOne) [reply website/public/index.ejs.html]: Ok, I _think_ I have this sorted out. I apologize for the craziness, I suppose maybe there is something in my editor that formats things differently. I'm still learning so it's been a struggle. Thanks for the guidance. Does this appear to be better?

**Review by vegerot — APPROVED** [review]
good work!

**Review by strager — APPROVED** [review]
This looks good to me.

Must fix: Write a better PR title.

**Comment by redconOne** [comment]
Thanks you two. I appreciate your time. I changed the PR title, is this more appropriate?

**Comment by strager** [comment]
Landed as commit 62a655afccd57a637db165b238253038dff761d4.

**Comment by vegerot** [comment]
p.s. I think after TypeScript support lands it should replace "Easy npm install"
